packages/sign_in_with_apple/lib/src/sign_in_with_apple.dart

```diff
@@ -183,15 +179,15 @@ class SignInWithApple {
   /// - Android
   ///
   /// In case Sign in with Apple is not available, the returned Future completes with `false`.
-  static Future<bool> isAvailable() {
-    return channel.invokeMethod<bool>('isAvailable');
+  static Future<bool> isAvailable() async {
+    return (await channel.invokeMethod<bool>('isAvailable')) ?? false;
   }
 
   static Future<AuthorizationCredentialAppleID> _signInWithAppleAndroid({
-    @required List<AppleIDAuthorizationScopes> scopes,
-    @required WebAuthenticationOptions webAuthenticationOptions,
-    @required String nonce,
-    @required String state,
+    @required required List<AppleIDAuthorizationScopes> scopes,
```

Comment:
I guess these can all be reduced to the single, new `required`, though I don't recall docs on it.

Reply:
Removed in 6221538df18cabf47069a1155fb660c09e60de88